network-api/networkapi/wagtailpages/templates/wagtailpages/blog_index_page.html

```diff
@@ -7,52 +7,82 @@
   <link rel="alternate" type="application/rss+xml" href="{% url 'rss-feed' %}" />
 {% endblock %}
 
+{% block index_header %}
+<div class="tw-grid tw-grid-cols-12">
+  <div class="tw-col-span-12 large:tw-col-span-8 tw-mt-4">
+    <h1 class="tw-h1-heading mb-0 mt-1 pt-2">
+      {% if index_title %}{{ index_title }}{% elif root.title %}{{ root.title }}{% elif page.header %}{{ page.header }}{% else %}{{ page.title }}{% endif %}
+    </h1>
+    <p class="tw-body-large mt-3 mb-0">{{ page.intro|safe }}</p>
+  </div>
+  <div class="tw-col-span-12 large:tw-col-span-4 tw-mt-4 large:tw-mt-6">
+    <form action="" method="get">
+      {% include "./fragments/blog_search_bar.html" with name="q" search_query=query %}
+    </form>
+</div>
+</div>
+
+<div class="row">
+  <div class="col-12">
+    <hr class="intro-and-content-divider">
+  </div>
+</div>
+{% endblock %}
 
 {% block featured %}
   {% if not filtered %}
-    <div class="tw-grid tw-grid-cols-12 tw-gap-x-5 tw-gap-y-7 tw-mb-7">
-      {%  for feature in page.specific.featured_pages.all %}
-        {% if forloop.first %}
-          <div class="tw-col-span-12">
-            {% include "./fragments/blog_index_feature.html" with blog_page=feature.blog %}
-          </div>
-        {% elif forloop.counter == 5 %}
-          <div class="tw-col-span-12 large:tw-col-span-7 large:tw-pb-7">
-            {% include "./fragments/blog_card.html" with page=feature.blog %}
-          </div>
-          <div class="tw-col-span-12 large:tw-col-span-5 xlarge:tw-pl-3">
-            {% for block in page.callout_box %}
-              {% include_block block %}
-            {% endfor %}
-          </div>
-        {% else %}
-          <div class="tw-col-span-12 large:tw-col-span-4">
-            {% include "./fragments/blog_card.html" with page=feature.blog %}
+    <div class="tw-grid tw-grid-cols-12 tw-gap-5 tw-mb-1 medium:tw-mb-7">
```

Comment:
@fessehaye After discussing with you, I decided to use "divider_classes" which adds some padding to the bottom and a 1px border, instead of using the tailwind divider as this is always going to be a fixed number of items and we can decide which do and do not get the divider.